appendix-basic65.tex

```diff
@@ -7462,12 +7458,9 @@ \subsection{RSPCOLOR}
 \item [Format:] {\bf RSPCOLOR(n)}
 \item [Usage:]  Returns multi-colour sprite colours.
 
-                {\bf n} 1 : get multi-colour \# 1.
-
-                {\bf n} 2 : get multi-colour \# 2.
+                {\bf n} the sprite colour to retrieve. Must be either 1, or 2.
```

Comment:
Probably restore these to the prior 'get multi-colour \# 1/2' terms, as I these terms relate more to the specifics of C64-style sprites that had a multi-colour mode.

So for monochrome sprites, they can have a single colour, defined via the `SPRITE` command's 'colour' parameter.

For multi-colour sprites, they can contains 3 colours:
* 1st colour is configurable per sprite, set via the `SPRITE` command
* 2nd and 3rd colours are global across all 8 sprites (can't be unique per sprite), and are set via the `SPRCOLOR` command.

So in the `SPRCOLOR` commands description, those two colours get given names like `mc1` and `mc2`, so I think it's best to use similar-sounding terms here too, to help cement the relationship to the reader.

Reply:
Restored + put them in a list so it looks similar to the other commands.

Reply:
Fine with the reversion here, just had a comment on the new line you added above it (`sprite number:`), and added a comment there.